src/main/java/duke/command/Command.java

```diff
@@ -0,0 +1,42 @@
+package duke.command;
+
+import java.io.IOException;
+
+import duke.History;
+import duke.ResponseFormatter;
+import duke.Storage;
+import duke.TaskList;
+
+
+/**
+ * Abstract command class that all commands inherits from
+ */
+public abstract class Command {
+    /**
+     * to denote when to exit the loop
+     */
+    protected boolean isExit = false;
+
+    /**
+     * get the exit check
```

Comment:
same for this javadoc, should start with caps and it should be a singular verb, i.e. `Gets`